bridge/docs/bridging.md

```diff
@@ -120,8 +120,8 @@ A refund on TFChain is initiated when either of the following conditions is met:
 
 We didn't mentioned yet a few TFChain event related to the flows discussed above, these events are:
 
-*   `tftBridgeModule.BurnTransactionExpired`
-*   `tftBridgeModule.RefundTransactionExpired`
+*   `tftBridgeModule.BurnTransactionExpired`: from TFCHAIN -> STELLAR burn flow, when transaction (burn TFT on TFChain source account and transfer amount from Stellar vault account to Stellar destination account) is stuck in bridge.
+*   `tftBridgeModule.RefundTransactionExpired`: from STELLAR -> TFCHAIN refund flow, after TFT were locked into Stellar vault, finally not minted to TFChain, and transaction to get it back to Stellar source account is stuck in bridge.
```

Comment:
I think saying `stuck in bridge` is technically wrong? you can say the transfer is pending approval from bridge daemons instead which more accurate  
also Both events explained below, if you think we need to include a short description here as well lets say something like
- certain number of TFChain blocks passed without a `BurnTransaction` being noticed and signed by the majority of bridge validators
- certain number of TFChain blocks passed without a `RefundTransaction` being noticed and signed by the majority of bridge validators

Reply:
Correct, I ll change